cypress/e2e/login_test/login_e2e.cy.js

```diff
@@ -0,0 +1,29 @@
+import { apiPath } from "../../../src/js/api/constants";
+
+it("logs in user and saves token using API request", () => {
+  // Authenticate using a POST request to the login API endpoint
+  cy.request({
+    method: "POST",
+    url: `${apiPath}/social/auth/login`,
+    body: {
+      email: "engangtil@stud.noroff.no",
+      password: "Tascam123",
+    },
+  }).then((res) => {
+    if (res.status === 200 && res.body.accessToken) {
+      // Store the access token in local storage
+      cy.window().then((win) => {
+        win.localStorage.setItem("token", res.body.accessToken);
```

Comment:
This isn't right, you shouldn't be setting the token yourself - but observing to see if the application successfully sets the token.